src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const remembe = people.filter(person => person.sex === 'm');
```

Comment:
```suggestion
  const man = people.filter(person => person.sex === 'm');
```
change to more meaningful name